packages/libs/src/api/Track.ts

```diff
@@ -583,6 +585,7 @@ export class Track extends Base {
     this.creatorNode.validateTrackSchema(metadata)
   }
 
+  // WARNING: CLIENT USES DIRECTLY IN upload/sagas.js !!!
```

Comment:
let's make this public.
I also wonder why we want to call it generateTrackId instead of getUnclaimedId?

Reply:
in retrospect, looks like I could call `libs.discoveryProvider.getUnclaimedId('tracks')` directly? Should I just do that

Reply:
yes!

Reply:
lol actually this method is only "private" by convention, the `_`. I removed that and just made it completely public, feels fine 🤷 Can revisit when we remove this entirely and move to SDK I think.

Reply:
ok cool